scrips/timeCalculations.js

```diff
@@ -18,4 +20,38 @@ const getTimeZone = (today) => {
     return `GMT${sign}${formattedTimeZone}`;
 };
 
-export { getStartOfWeek, getTimeZone };
+const minutesToHour = (minutes) => {
+    return Math.round((parseInt(minutes, 10) / 60) * 100) / 100;
+};
+
+const hoursToMinutes = (hours) => {
+    return Math.round(hours * 60);
+};
+
+const formatHours = (hour) => (parseInt(hour, 10) < 10 ? `0${parseInt(hour, 10)}` : hour);
+
+const formatMinutes = (minutes) => (parseInt(minutes, 10) < 10 ? `0${parseInt(minutes, 10)}` : minutes);
+
+const displayTime = (hour, minutes) => {
+    return `${formatHours(hour)}:${formatMinutes(minutes)}`;
+};
+
+const formatTime = (hourFraction) => {
+    let hour = Math.floor(hourFraction);
+    let minutes = hourFraction - hour;
+
+    if (hour >= hoursNumber) hour = hour - hoursNumber;
+
+    return displayTime(hour, hoursToMinutes(minutes));
```

Comment:
You  `minutes` to hoursToMinutes 
```suggestion
    return displayTime(hour, hoursToMinutes(minutes));
```